cli/dcoscli/data/help/node.txt

```diff
@@ -50,6 +57,14 @@ Options:
         Agent node with the provided ID.
     --user=<user>
         The SSH user, where the default user [default: core].
+    --list
+        List available snapshots.
+    --status
+        Print snapshot job status.
+    --cancel
+        Cancel a running snapshot job.
+    --download=<snapshot>
+        Download a snapshot to current directory.
     --version
         Print version information.
 
```

Comment:
You'll need to also update here: https://github.com/dcos/dcos-cli/blob/master/cli/tests/data/help/node.txt